.travis.yml

```diff
@@ -0,0 +1,25 @@
+sudo: false
+language: node_js
+cache:
+  directories:
+    - node_modules
+node_js:
+  - '6'
+before_install:
+  - npm i -g npm@^3.0.0
+  - npm i -g surge
+  - npm i -g @schibstedspain/sui-studio
```

Comment:
It may not be something to worry about. But you could cache global node_modules for a little speed bump.
But anyway better wait to see the current times and compare if you will